drivers/wifi/wfx200/wfx200_internal.h

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) 2021 grandcentrix GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __ZEPHYR_DRIVERS_WIFI_SILABS_WFX200_INTERNAL_H_
+#define __ZEPHYR_DRIVERS_WIFI_SILABS_WFX200_INTERNAL_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <drivers/spi.h>
+#include <net/ethernet.h>
+#include <net/wifi_mgmt.h>
+
+#include <sl_wfx_constants.h>
+
+enum wfx200_event {
+	WFX200_CONNECT_EVENT = 0,
+	WFX200_CONNECT_FAILED_EVENT,
+	WFX200_DISCONNECT_EVENT,
+	WFX200_AP_START_EVENT,
+	WFX200_AP_START_FAILED_EVENT,
+	WFX200_AP_STOP_EVENT,
+};
+
+struct wfx200_queue_event {
+	enum wfx200_event ev;
+	union {
+
+	};
+};
+
+struct wfx200_gpio {
+	const struct device *dev;
+	uint8_t pin;
+};
```

Comment:
This looks like it should be a gpio_dt_spec.

Reply:
:heavy_check_mark: